File: Sokoban 2.0 Consol Tester2/Sokoban 2.0 Consol Tester/Control/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Sokoban_2._0_Consol_Tester
{
    public class Player
    {

        private string name;
        private ControlInterface controlInterface;

        // Constructor for the player
        public Player(string name)
        {
            this.name = name;
        }

        public void lost()
        {
        }

        public void won()
        {
        }

        // Adds the given control interface to the player
        public void AddControl(ControlInterface controlInterface)
        {
            this.controlInterface = controlInterface;
        }

        public ControlInterface GetCI()
        {
            return controlInterface;
        }

        public string GetName()
        {
            return name;
        }
    }
}

File: sokoban/src/hu/bme/sch/sokoban/game/Player.java
package hu.bme.sch.sokoban.game;

import hu.bme.sch.sokoban.game.controls.ControlInterface;

public class Player {
    private String name;
    private ControlInterface controlInterface;

    // Constructor for the player
    public Player(String name){
        this.name = name;
    }

    public void lost() {
    }

    public void won() {
    }

    // Adds the given control interface to the player
    public void addControl(ControlInterface controlInterface) {
        this.controlInterface = controlInterface;
    }
    
    public ControlInterface getCI() {
    	return controlInterface;
    }

    public String getName() {
        return name;
    }
}
